frontend/src/container/MetricsApplication/MetricsPageQueries/DBCallQueries.ts

```diff
@@ -12,35 +12,42 @@ export const databaseCallsRPS = ({
 	legend,
 	tagFilterItems,
 }: DatabaseCallsRPSProps): QueryBuilderData => {
-	const metricName: BaseAutocompleteData = {
-		dataType: 'float64',
-		isColumn: true,
-		key: 'signoz_db_latency_count',
-		type: null,
-	};
+	const metricNames: BaseAutocompleteData[] = [
+		{
+			dataType: 'float64',
```

Comment:
a lot of magic strings are here, please make constant for same

Reply:
Sure